runtime/src/main/java/org/corfudb/runtime/view/workflows/WorkflowRequest.java

```diff
@@ -59,20 +59,19 @@
      * orchestrator
      */
     protected ManagementClient getOrchestrator(@NonNull Layout layout) {
-        List<String> activeLayoutServers = layout.getLayoutServers().stream()
-                .filter(s -> !layout.getUnresponsiveServers().contains(s)
-                        && !s.equals(nodeForWorkflow))
+        List<String> availableLayoutServers = layout.getLayoutServers().stream()
+                .filter(s -> !s.equals(nodeForWorkflow))
```

Comment:
Would this approach degrade the performance of workflow since we `pingSync` the endpoint everytime before we select it? Maybe it's better to first try to getOrchestrator from the responsiveServers as the layout suggests and if none of them are actually responsive, then we try to select from the unresponsiveServers?